File: C++/BruteForce.cpp
#include <iostream>

//takes in a number and returns true if it is prime and false otherwise, worst possible solution with O(N) time notation
bool isPrime(long long num) {
    if (num <= 1) return false;
    if (num == 2) return true;
    for (long long i = 2; i<num; i++) {
        if (!(num%i)) return false;
    }
    return true;
}


int main() {
    long long num;
    std::cout << "Enter a number to check if it's prime or not.\n";
    std::cin >> num;
    std::cout << "The number you entered is "<<(isPrime(num) ? "prime" : "not prime")<< "\n";
    return 0;
}

File: Javascript/BruteForce.js
//brute force, O(n) time notation
function isPrime(num) {
    if (num <= 1) return false;
    if (num==2) return true;
    for (let i = 2; i<num; i++) {
        if (num%i==0) return false;
    }
    return true;
}

const test = 10; // number to check
console.log("The number "+test+" is "+(isPrime(test)?"prime":"not prime"));
